helm/svc-bgs-api/values.yaml

```diff
@@ -54,3 +54,9 @@ bip:
   # TODO: Consider moving these URL settings to application-*.yml files
   bipClaimUrl:    claims-uat.stage.bip.va.gov/api/v1
   bipEvidenceUrl: vefs-claimevidence-uat.stage.bip.va.gov/api/v1/rest
+
+secrets:
+  rabbitmq:
+    name: vro-dd-rabbitmq-pass
+    key: rabbitmq_pass
```

Comment:
it looks like `secrets.rabbitmq.name` and `secrets.rabbitmq.key` are consistent, regardless of environment, is that correct? if so, then I'd recommend removing the overrides in the env-specific files (eg `values-for-sandbox.yaml`) in this case.

Reply:
Yes but when we had it in the deployment.yaml it wasn't being pulled/mounted in other environments.